pktsetup/pktsetup.c

```diff
@@ -27,14 +27,22 @@
 #include <sys/stat.h>
 #include <unistd.h>
 #include <getopt.h>
+#if defined(__linux__)
 #include <bits/types.h>
+#else
+#include <mach/mach_types.h>
+#include "../include/darwin/types.h"
+#endif
 #include <sys/types.h>
 #include <string.h>
 #include <limits.h>
 #include <stdlib.h>
 #include <errno.h>
-
+#if defined(__linux__)
 #include <linux/cdrom.h>
+#else
+#include "darwin/cdrom.h"
+#endif
```

Comment:
Are you sure that your Darwin system implement pktcdvd interface from Linux kernel?

Reply:
Not sure about the actual working of everything in this PR: with these patches it all compiles, and runs (tried out all binaries with help/version switches). Of course, (possibly missing) functionality is to be tested (I'm willing to do it going forward, of course).